src/main/java/embl/ebi/variation/eva/pipeline/gene/FeatureCoordinates.java

```diff
@@ -32,21 +32,21 @@
     public FeatureCoordinates() {
     }
 
-    public FeatureCoordinates(String featureId, String name, String feature, String chromosome, int start, int end) {
-        this.featureId = featureId;
+    public FeatureCoordinates(String id, String name, String feature, String chromosome, int start, int end) {
+        this.id = id;
```

Comment:
it is automatically picked as `_id`, here there is an example document: 
```
{
	"_id" : "ENSCSAG00000017073",
	"_class" : "embl.ebi.variation.eva.pipeline.gene.FeatureCoordinates",
	"name" : "FBXO25",
	"feature" : "gene",
	"chromosome" : "8",
	"start" : 183180,
	"end" : 246703
}
```
there's a test that requests the indexes, and this is what it gets:
```
[{ "v" : 1 , "key" : { "_id" : 1} , "name" : "_id_" , "ns" : "variants.features"}, { "v" : 1 , "key" : { "name" : 1} , "name" : "name_1" , "ns" : "variants.features" , "sparse" : true , "background" : true}]
```
and _id [is unique by default](https://docs.mongodb.com/manual/indexes/#default-id-index)